package.json

```diff
@@ -4,16 +4,21 @@
   "private": true,
   "dependencies": {
     "@blueprintjs/core": "^2.2.1",
-    "node-sass": "^4.9.0",
+    "node-sass-chokidar": "1.3.0",
```

Comment:
I think I owe some context on why we weren't using `node-sass-chokidar` to begin with.

In 8876362, I upgraded the version of `react-scripts` we use so that we can take advantage of improvements in v2 of that package. 

The full list of improvements is described (in this GH issue)[https://github.com/facebook/create-react-app/issues/3815], but the main reason we sprung for this is for the [upgraded webpack configuration.](https://github.com/facebook/create-react-app/pull/4195/files#diff-d2bddbd3bfb7051f9381474c844674faR337)

However, because I made a mistake in pushing that code directly to master (I intended to branch, but my night-brain didn't comply), I didn't sufficiently explain what was happening. I can see why one would go this route without that context.

The stable version of `react-scripts` (which is currently referenced in the `README` for the master branch of `create-react-app`) doesn't include any of these preprocessors as peer dependencies, which is why they recommend the use of a separate process to build styles than the rest of your app. The version we're using as of 8876362 does include those preprocessors, which is why we don't need to install `node-sass-chokidar` or add these separate build scripts.

That said, I know you mentioned you were running into issues when trying to use this `react-scripts@2` as-is. A few things I'd recommend checking out after reverting the addition of `node-sass-chokidar`, but keeping the SCSS additions intact:

- Clear your browser cache
- Remove and reinstall all packages, then run `yarn start`

If this doesn't cut it, we might want to consider [removing the service worker](https://github.com/mplsjrdevs/mplsjrdevs/blob/master/src/index.js#L8) that CRA adds by default. As this is essentially a static site, having the service worker present seems unnecessary.

Let me know if I can add more detail around the above. I know this is kind of a wall-of-text, trying to get all this on paper before standup :)

Reply:
Per our internal discussion, I'm no longer of the opinion that my above message tracks the right way to go. Let's ship this! Thanks, @jenessawhite!